docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:latest
```

Comment:
better to set a fixed version, 22.04 ?

Reply:
Yes right. @nicoloboschi  I also added a simple docker-compose file to run a 1 node carapace cluster with zookeeper and bookkeeper/herddb embedded.

This can be usefull for test.